File: JavaAbril30dias/src/Dia10.java
import java.util.Scanner;

public class Dia10 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int boxPrice = 6;
        System.out.println("¿Cuantas cajas quieres?");
        int boxQuantity = input.nextInt();
        double totalPrice = boxPrice * boxQuantity;
        if (boxQuantity >= 5) {
            if (boxQuantity > 15) {
                System.out.println("Gastos de envío: 0€.");
            }
            else {
                System.out.println("Gastos de envío: 10€.");
            }
            if (totalPrice > 250) {
                double discountPrice = Math.round((totalPrice * 0.9) * 100.0) / 100.0;
                System.out.println("Precio final con 10% de descuento: " + discountPrice);
            }
            else if (totalPrice > 120) {
                double discountPrice = Math.round((totalPrice * 0.95) * 100.0) / 100.0;
                System.out.println("Precio final con 5% de descuento: " + discountPrice);
            }
            else {
                System.out.println("Precio final sin descuento: " + totalPrice);
            }
        }
        else {
            System.out.println("No puedes comprar más de 5 cajas.");
        }
        input.close();
    }
}

File: PythonAbril30dias/dia10.py
box_price = 6
box_quantity = int(input("¿Cuantas cajas quieres?"))
total_price = 6 * box_quantity

if box_quantity >= 5:
    if box_quantity < 15:
        print("Gastos de envío: 10€.")
    else:
        print("Gastos de envío: 0€.")
    if 120 < total_price < 250:
        discount_price = round(total_price * 0.95, 2)
        print(f"El precio final es {discount_price}")
    elif total_price > 250:
        discount_price = round(total_price * 0.9, 2)
        print(f"El precio final es {discount_price}")
    else:
        print(f"El precio final es {total_price}")
else:
    print("No puedes comprar menos de 5 cajas.")